templates/zerver/emails/password_reset_non_existent_user.subject

```diff
@@ -0,0 +1 @@
+Password reset requested for your Zulip account
```

Comment:
Are we using this?

Reply:
No, that's from the original implementation (that added a new template) and should be deleted.

Reply:
Fixed.